libs/net/dal/Services/ContentService.cs

```diff
@@ -220,6 +220,9 @@ public async Task<IPaged<Content>> FindWithElasticsearchAsync(ContentFilter filt
         if (!string.IsNullOrWhiteSpace(filter.Headline))
             filterQueries.Add(s => s.Wildcard(m => m.Field(p => p.Headline).Value($"*{filter.Headline.ToLower()}*")));
 
+        // if (!string.IsNullOrWhiteSpace(filter.Keyword))
```

Comment:
You haven't figured out how to make this request yet correct?  If so add a `// TODO: description`